src/arrayMethodSort.js

```diff
@@ -3,9 +3,42 @@
 /**
  * Implement method Sort
  */
+
+function sortByString(a, b) {
```

Comment:
This function is not sorting, it's comparing.
```suggestion
function compareByString(a, b) {
```